images/k8s-cloud-builder/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN apt-get -qqy purge ".*python2.*" \
 # Install packages
 RUN apt-get install -qqy \
         apt-transport-https \
+        bsdmainutils \
```

Comment:
Out of interest: why? what are we currently missing?

Generally, if we find ourselves adding many things for the image meant for "interactive" use, I'd opt for creating a different image or at least a specific tag for the "interactive" image, e.g. `k8s-release-interactive` or `k8s-cloud-builder:latest-interactive` or that like. I don't want to be super strict when it comes to bring in new stuff to `k8s-cloud-builder` but I also don't wanna just open up the floodgates.

Reply:
The gcbmgr tool complained that the dependency is not available when running it. :man_shrugging: